src/components/redditClone/frontPage/styles.scss

```diff
@@ -0,0 +1,48 @@
+.frontpagecontainer {
```

Comment:
the formatting of this is off (look at your `}`'s). This should be automatic and should fail the build so it's my fault.